wordpress-updatable/README.md

```diff
@@ -0,0 +1,66 @@
+*This directory contains an example Kubernetes application (app) based on
+WordPress for the purpose of demonstrating app integration with
+Google Cloud Marketplace. **Not intended for actual use!***
+
+*Content below is intended as a template for end-user documentation. Work in
+progress.*
+
+# Overview
+
+WordPress is a free and open-source content management system (CMS) based on PHP
+and MySQL...
+
+# Installation
+
+## Quick install with Google Cloud Marketplace
+
+Get up and running with a few clicks! Install this WordPress app to a
+Google Kubernetes Engine cluster using Google Cloud Marketplace. Follow the
+on-screen instructions:
+*TODO: link to solution details page*
```

Comment:
Please link all TODOs to an issue so we can track them.

Reply:
This README was copied as-is. Opened #124 for cleaning these up.